src/UserAdmin/src/Smart.FA.Catalog.Core/Domain/IEFSoftDelete.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace Smart.FA.Catalog.Core.Domain;
+
+/// <summary>
+/// Add Soft Delete field to an entity
+/// </summary>
+public interface IEFSoftDelete
```

Comment:
I would suggest adding the full name of EF here for clarity.
```suggestion
public interface IEntityFrameworkSoftDelete
```

Reply:
On second thought, shouldn't it be called `IEntitySoftDelete`?

Reply:
Yes I am not too fond of my name either retrospectively. Why not IEntitySoftDelete indeed but what do you say about simplifying it to `ISoftDelete `?

Reply:
Yes, I was think ing about that after my remark as well :D
Make it `ISoftDelete`